### PR TITLE
Update gRPC BBEs with metatags

### PR DESCRIPTION
**Pull request by daneshk**
## Purpose
$subject

### CODE REVIEW

File: examples/grpc-bidirectional-streaming/grpc_bidirectional_streaming.metatags [hunk 1]
```diff
@@ -0,0 +1,2 @@
+description: This BBE has a gRPC streaming service and a non-blocking streaming client. Both client and server send stream of messages to each other.
```
Comment (Samallama) [inline examples/grpc-bidirectional-streaming/grpc_bidirectional_streaming.metatags:1]:
```suggestion
description: This Ballerina by Example has a gRPC streaming service and a non-blocking streaming client. Both the client and the server sends a stream of messages to each other.
```

File: examples/grpc-client-streaming/grpc_client_streaming.metatags [hunk 2]
```diff
@@ -0,0 +1,2 @@
+description: This BBE has a gRPC service and a non-blocking streaming client. The client sends stream of request messages and gets a response message.
```
Comment (Samallama) [inline examples/grpc-client-streaming/grpc_client_streaming.metatags:1]:
```suggestion
description: This Ballerina by Example has a gRPC service and a non-blocking streaming client. The client sends a stream of request messages and gets a response message.
```

File: examples/grpc-secured-unary/grpc_secured_unary.metatags [hunk 3]
```diff
@@ -0,0 +1,2 @@
+description: This BBE has a gRPC unary service and a blocking client. The client sends and receives messages in a secured connection.
```
Comment (Samallama) [inline examples/grpc-secured-unary/grpc_secured_unary.metatags:1]:
```suggestion
description: This Ballerina by Example has a gRPC unary service and a blocking client. The client sends and receives messages in a secured connection.
```

File: examples/grpc-secured-unary/grpc_secured_unary.metatags [hunk 4]
```diff
@@ -0,0 +1,2 @@
...
+keywords: ballerina, ballerina by example, bbe, grpc, unary, simple, blocking, secure connection, tls
```
Comment (Samallama) [inline examples/grpc-secured-unary/grpc_secured_unary.metatags:2]: Shall we remove one keyword? The best practice is to have 5-8 keywords

File: examples/grpc-server-streaming/grpc_server_streaming.metatags [hunk 5]
```diff
@@ -0,0 +1,2 @@
+description: This BBE has a gRPC streaming service and a non-blocking client. The client sends a request to the server and receives a stream of messages.
```
Comment (Samallama) [inline examples/grpc-server-streaming/grpc_server_streaming.metatags:1]:
```suggestion
description: This Ballerina by Example has a gRPC streaming service and a non-blocking client. The client sends a request to the server and receives a stream of messages.
```

File: examples/grpc-unary-blocking/grpc_unary_blocking.metatags [hunk 6]
```diff
@@ -0,0 +1,2 @@
+description: This BBE has a gRPC unary service and a blocking client. The client sends and receives messages in blocking manner.
```
Comment (Samallama) [inline examples/grpc-unary-blocking/grpc_unary_blocking.metatags:1]:
```suggestion
description: This Ballerina by Example has a gRPC unary service and a blocking client. The client sends and receives messages in a blocking manner.
```

File: examples/grpc-unary-non-blocking/grpc_unary_non_blocking.metatags [hunk 7]
```diff
@@ -0,0 +1,2 @@
+description: This BBE has a gRPC unary service and a non-blocking client. The client sends and receives messages in non-blocking manner.
```
Comment (Samallama) [inline examples/grpc-unary-non-blocking/grpc_unary_non_blocking.metatags:1]:
```suggestion
description: This Ballerina by Example has a gRPC unary service and a non-blocking client. The client sends and receives messages in a non-blocking manner.
```

File: examples/grpc-secured-unary/grpc_secured_unary.metatags [hunk 8]
```diff
@@ -0,0 +1,2 @@
+description: This Ballerina by Example has a gRPC unary service and a blocking client. The client sends and receives messages in a secured connection.
+keywords: ballerina, ballerina by example, bbe, grpc, unary, simple, blocking, secure connection, tls
```
Comment (daneshk) [inline examples/grpc-secured-unary/grpc_secured_unary.metatags:2]:
```suggestion
keywords: ballerina, ballerina by example, bbe, grpc, unary blocking, secure connection, tls
```

**Pull request merged**